pkg/capi/edit/cluster.x-k8s.io.cluster/ClusterConfig.vue

```diff
@@ -42,9 +41,7 @@ interface Step {
   name: String
 }
 
-export default (Vue as VueConstructor<
-  Vue & InstanceType<typeof CreateEditView>
->).extend({
+export default {
```

Comment:
Can this use defineComponent instead? I believe we lose some vue component typing with a plain `export default` in components